Life.java

```diff
@@ -25,31 +29,94 @@ public Life(String[] setup) {
 
   @Override
   public void nukeAll() {
-    // TODO Auto-generated method stub
-
+    //Alle löschen
+    raster = new boolean[10][10];
+    
   }
 
   @Override
   public void setAlive(int x, int y) {
-    // TODO Auto-generated method stub
+    raster[x][y] = true; //Zellen sind da
 
   }
 
   @Override
+
   public void setDead(int x, int y) {
-    // TODO Auto-generated method stub
+    //Zelle an x, y ist "falsch" -> tot
+    raster[x][y] = false;
 
   }
 
   @Override
   public boolean isAlive(int x, int y) {
-    // TODO Auto-generated method stub
-    return false;
+    //Gucken ob Zelle im Raster ist
+    return raster[x][y];
+    
   }
 
   @Override
   public ILife nextGeneration() {
-    // TODO Auto-generated method stub
-    return null;
+   
+   boolean[][] future = new boolean[raster.length][raster[0].length];
```

Comment:
Anstelle die Folgegeneration innerhalb des Objekts zu berechnen wäre es "moderner", ein neues Life-Objekt zu erzeugen und dieses (nicht this) zurückzugeben.